### PR TITLE
Make more blocks responsive

### PR DESCRIPTION
**Pull request by 102**

### CODE REVIEW

File: frontend/blocks/array/array_field.css [hunk 1]
```diff
@@ -1,6 +1,14 @@
 @import "../../styles/_variables.css";
 
 .$(cls-prefix)-form-array {
+  .remove-array-item {
+    padding: 6px 0;
+
+    @media (min-width: 768px) {
```
Comment (abukurov) [inline frontend/blocks/array/array_field.css]: move into variable

File: frontend/blocks/fieldset/fieldset.css [hunk 2]
```diff
@@ -20,12 +20,13 @@
...
       border: 1px solid $border-clr;
       color: #919191;
-      display: none;
+      display: block;
       font-size: 15px;
       font-weight: 400;
       height: 42px;
       margin: 0;
       padding: 11px 12px 0;
+      overflow: auto;
```
Comment (abukurov) [inline frontend/blocks/fieldset/fieldset.css]: you can omit this style

File: frontend/blocks/fieldset/fieldset.css [hunk 3]
```diff
@@ -40,7 +41,8 @@
     }
 
     .control-label.font-size-small {
-      display: none;
+      display: block;
```
Comment (abukurov) [inline frontend/blocks/fieldset/fieldset.css]: you have already set this style above

File: frontend/blocks/fieldset/fieldset.css (hunk 2 above)
Comment (102) [inline frontend/blocks/fieldset/fieldset.css]: Default values (at least from chrome):
```css
overflow-x: visible;
overflow-y: visible;
```

**Pull request merged**